student.rb

```diff
@@ -1,14 +1,12 @@
 require_relative 'person'
 
 class Student < Person
-  attr_accessor :classroom
-
-  def initialize(name, age, classroom, parent_permission: true)
-    super(name, age, parent_permission: parent_permission)
-    @classroom = classroom
+  def initialize(age, class_room, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @class_room = class_room
   end
 
   def play_hooky
-    '¯\\(ツ)/¯'
+    '¯\(ツ)/¯'
   end
```

Comment:
- [ ] With the association between `Classroom` and `Student` you would have a method where a student is added to the classrooms' students. Let's add it

Reply:
- [ ] @OpondoG as previously requested kindly add a method to add a student to the classroom to achieve the belongs-to association.